src/main/java/pl/javastart/task/QueueManager.java

```diff
@@ -0,0 +1,131 @@
+package pl.javastart.task;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.Scanner;
+
+public class QueueManager {
+    private Queue<Vehicle> vehicleQueue = new LinkedList<>();
+    private Scanner sc = new Scanner(System.in);
+    private File queueFile = new File("src/main/resources/queue.txt");
+
+    void mainLoop() throws IOException {
+        Queue<Vehicle> vehicleYesterdayQueue;
+        vehicleYesterdayQueue = ReadWriteData.readFile(queueFile);
+        if (!vehicleYesterdayQueue.isEmpty()) {
+            vehicleQueue = vehicleYesterdayQueue;
+        }
```

Comment:
To można skrócić do jednego wiersza `vehicleQueue = ReadWriteData.readFile(queueFile)`
Ostateczny efekt będzie dokładnie taki sam